src/modbus.c

```diff
@@ -573,6 +573,8 @@ static int check_confirmation(modbus_t *ctx, uint8_t *req,
         }
 
         /* Check the number of values is corresponding to the request */
+        int resp_addr_ok = TRUE;
+        int resp_data_ok = TRUE;
         switch (function) {
```

Comment:
I don't like the declaration at this point, can we move it to beginning of function?

Reply:
@mhei I moved the declaration to the beginning of the block where the other variables used inside this block are also defined.